packages/vscode/package.json

```diff
@@ -215,13 +228,15 @@
         },
         "astro.format.indentFrontmatter": {
           "type": "boolean",
-          "default": true,
-          "title": "Formatting: Indent the frontmatter by one level of indentation"
+          "default": false,
```

Comment:
It was decided in #ecosystem on Discord that this one should be turned off by default